engines/order_management/spec/controllers/order_management/reports/enterprise_fee_summaries_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "spec_helper"
+
+describe OrderManagement::Reports::EnterpriseFeeSummariesController, type: :controller do
```

Comment:
I changed this at the same timee as I moved the file (should have been a separate commit): the namespace here was Spree::Admin::Reports